client/src/views/pages/banned.js

```diff
@@ -0,0 +1,556 @@
+  const history = useHistory();
+
+  return (
+    <>
+      <Row className="text-center">
+        <Col>
+          <h3>Who are you?</h3>
+          <p>
+            Please enter your Steam 64 ID in the form below so we can tailor the following
+            information to you. You can find your Steam 64 ID{' '}
+            <a href="https://steamidfinder.com/lookup/" target="_blank" rel="noreferrer">
+              here
+            </a>
+            . Once you have entered your Steam 64 ID click "Next".
+          </p>
+          <FormGroup>
+            <InputGroup
+              className={classnames({
+                'is-invalid': steamID !== '' && !(steamID && steamID.match(/^[0-9]{17}$/))
+              })}
+            >
+              <InputGroupAddon addonType="prepend">
+                <InputGroupText>
+                  <i className="fa fa-search" />
+                </InputGroupText>
+              </InputGroupAddon>
+
+              <Input
+                type="text"
+                placeholder="Steam 64 ID"
+                value={steamID}
+                onChange={(e) => setSteamID(e.target.value)}
+              />
+            </InputGroup>
+            <FormFeedback>A valid Steam 64 ID is a 17 digit number.</FormFeedback>
+          </FormGroup>
+        </Col>
+      </Row>
+      <Row className="mt-4">
+        <Col className="text-left">
+          <Button color="default" disabled>
+            <i className="fa fa-arrow-left mr-2" />
+            Back
+          </Button>
+        </Col>
+        <Col className="text-center">
+          <Button onClick={props.nextStep}>Skip</Button>
+        </Col>
+        <Col className="text-right">
+          <Button
+            color="primary"
+            onClick={() => history.push(`/banned/${steamID}`)}
+            disabled={!(steamID && steamID.match(/^[0-9]{17}$/))}
+          >
+            Next
+            <i className="fa fa-arrow-right ml-2" />
+          </Button>
+        </Col>
+      </Row>
+    </>
+  );
+}
+
+function Foreword(props) {
+  return (
+    <>
+      <Row>
+        <Col>
+          <h3 className="text-center">Foreword</h3>
+          <p>
+            Hello <strong>{props.steamUser?.name || ''}</strong>,
+          </p>
+          <p>
+            If you have been directed to this page then it likely means that you are listed on and
+            potentially banned by the Squad Community Ban List.
+          </p>
+          <p>
+            The Squad Community Ban List is dedicated to helping to create a healthy Squad community
+            that everyone can enjoy, including players that were previous harmful to the community
+            and have since reformed. Therefore, we are keen to assist players in being able to
+            access the Squad community without the Squad Community Ban List limiting where they can
+            and cannot play. However, there are limits in what we can do to assist these players due
+            to our strict remit.
```

Comment:
I had to lookup remit, and I doubt I'm the only one that doesn't know it's meaning...

```suggestion
            to our strict remit.
```